src/main/java/org/opentripplanner/middleware/utils/ConfigUtils.java

```diff
@@ -18,6 +22,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(ConfigUtils.class);
 
     public static final String DEFAULT_ENV = "configurations/default/env.yml";
+    public static final String DEFAULT_ENV_SCHEMA = "configurations/default/env.schema.json";
```

Comment:
Nit: This can probably just be called `ENV_SCHEMA` since there should only ever be one of them.

Reply:
Updated.

Reply:
I think this file should be embedded as a resource (we don't want implementers to modify it).